src/components/DMsList/DMsListStyled.js

```diff
@@ -62,24 +92,48 @@ export const BadgeStyled = styled.div`
 export const ChatName = styled.div`
   display: flex;
   flex-direction: column;
-  justify-content: space-between;
-  width: 116px;
+  justify-content: center;
+  gap: 8px;
+  width: calc(100% - 58px);
   height: 50px;
-  & > p {
+`;
+
+export const NameAndDayBox = styled.div`
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+
+  p {
+    font-size: 14px;
+    font-weight: 400;
+    flex-shrink: 0;
     color: var(--color-grey-9);
+    white-space: nowrap;
   }
-  @media ${device.laptop} {
-    width: 156px;
-  }
 `;
 
-export const MessageDay = styled.div`
+export const CompanionName = styled.h6`
+  flex-grow: 1;
+  margin-right: 8px;
+  font-size: 18px;
```

Comment:
Let's move it into GlobalStyles. Currently we have there H5 and H6 tags with same font-size: 16px, I think we need to change font-size from 16px to 18px for the h6 tag.

Reply:
done. change font-size in globalStyles